core/trino-main/src/main/java/io/trino/metadata/SignatureBuilder.java

```diff
@@ -77,7 +77,7 @@ public SignatureBuilder longVariableConstraints(List<LongVariableConstraint> lon
 
     public SignatureBuilder argumentTypes(TypeSignature... argumentTypes)
     {
-        return argumentTypes(asList(requireNonNull(argumentTypes, "argumentTypes is Null")));
+        return argumentTypes(asList(requireNonNull(argumentTypes, "argumentTypes are null")));
```

Comment:
The previous message is correct. It's meant to read as "The 'argumentTypes' variable is null". The new message is ambiguous: is the variable null or are the elements all null?

Reply:
I agree. the plural form must not be used. A variable refers to a single being (an object), whether it's a Collection or not.